modules/exploits/windows/http/easy_file_sharing_web_server.rb

```diff
@@ -0,0 +1,195 @@
+##
+## This module requires Metasploit: http://metasploit.com/download
+## Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::Remote::Tcp
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+                      'Name'           => 'Easy File Sharing Web Server Remote Code Execution',
+                      'Description'    => %q{
+        # Easy File Sharing Web Server v6.8
```

Comment:
If you please, load the module in msfconsole, and type info. You will see that this description format doesn't work so well (unfortunately).